autograph/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM mozilla/autograph:latest
```

Comment:
Shouldn't we pin it here? As `2.1.1` is hard coded in the version.json, I think I would prefer

Reply:
Done. But I noticed that 2.1.1 is 3 versions (and 2 months) behind. 
https://hub.docker.com/r/mozilla/autograph/tags/
What matters the most actually is what version is used in cloudops-deployment.